src/app/search/SearchBar.m.scss

```diff
@@ -13,6 +13,8 @@
 .open {
   border-bottom-left-radius: 0 !important;
   border-bottom-right-radius: 0 !important;
+
+  box-shadow: inset 0 0 0 1px var(--theme-search-dropdown-border);
```

Comment:
This adds a border around the combined search box + dropdown when it's open. 

![image](https://github.com/DestinyItemManager/DIM/assets/1396158/c246be2a-6f2c-450f-86cb-2f291d34237f)

This makes it clear where there boundaries of the dropdown are since in this theme there is low contrast between the search dropdown and the layers behind it — this is also exaggerated on monitors with poor contrast. 

The `calc()` referenced in the previous comment is to offset the dropdown so the overall border is drawn around the combined search box + dropdown and to avoid showing the dividing line between them. 

This is how it looks without the 1px offset removed from `top` (unwanted dividing line between the 2 components): 
![image](https://github.com/DestinyItemManager/DIM/assets/1396158/677e7cd8-17bb-4c7e-8980-c5bd8ad4b64a)

I'll add a comment explaining what the calc is for